utils/extractor.py

```
@@ -1,10 +1,13 @@
     """
-    article = Article(url)
-    article.download()
-    article.parse()
-    return article.text
+    try:
+        article = Article(url)
+        article.download()
+        article.parse()
+        return article.text
+    except ArticleException:
+        raise ValueError("url 형식을 다시 확인해 주세요")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)